packages/debugger/package.json

```diff
@@ -17,9 +17,9 @@
   "scripts": {
     "build": "webpack --config webpack/webpack.config.js",
     "prepare": "yarn build",
-    "test": "mochapack --webpack-config webpack/webpack.config-test.js --recursive",
-    "test:coverage": "nyc mochapack --webpack-config webpack/webpack.config-test.js --recursive",
-    "test:debug": "NODE_ENV=testing node --inspect ./node_modules/.bin/mochapack --webpack-config webpack/webpack.config-test.js --recursive"
+    "test": "mochapack --webpack-config webpack/webpack.config-test.js \"./test/*.js\" \"./test/data/*.js\"",
+    "test:coverage": "nyc mochapack --webpack-config webpack/webpack.config-test.js \"./test/*.js\" \"./test/data/*.js\"",
+    "test:debug": "NODE_ENV=testing node --inspect ./node_modules/.bin/mochapack --webpack-config webpack/webpack.config-test.js \"./test/*.js\" \"./test/data/*.js\""
```

Comment:
What's the difference between `--recursive` and glob?

Reply:
`--recursive` runs its algorithm on all directories. We don't want it going and trying to run on the `sources` directory because it causes errors. Those fixtures are not meant for mochapack consumption.

Reply:
oh, cool! thanks!